tests/aead_tests.rs

```diff
@@ -509,6 +509,38 @@ fn test_aead_key_debug() {
     );
 }
 
+#[test]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+fn test_aead_lesssafekey_clone() {
```

Comment:
Please parameterize this on the `&'static aead::Algorithm` and then test it with all three AES-128-GCM, AES-256-GCM, and ChaCha20-Poly1305. Especially the AES and ChaCha20-Poly1305 code is completely different, it's good to cover all.